gw_eccentricity/eccDefinition.py

```diff
@@ -310,9 +340,21 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
             get_default_spline_kwargs(),
             "spline_kwargs",
             "utils.get_default_spline_kwargs()")
+        self.rational_fit_kwargs = check_kwargs_and_set_defaults(
+            self.extra_kwargs["rational_fit_kwargs"],
+            get_default_rational_fit_kwargs(),
+            "rational_fit_kwargs",
+            "utils.get_default_rational_fit_kwargs()")
         self.available_averaging_methods \
             = self.get_available_omega_gw_averaging_methods()
         self.debug_level = self.extra_kwargs["debug_level"]
+        # set verbose to debug_level. If verbose is True, then it prints information
+        # of each iteration for rational fits for a given degree.
+        self.rational_fit_kwargs["verbose"] = self.debug_level
```

Comment:
Why not just call it debug_level?

Reply:
"verbose" is the keyword used in the routine for rational fit, so I have kept that the same and instead assign value to it using `debug_level`